process/due-diligence-guidelines.md

```diff
@@ -0,0 +1,142 @@
+  CNCF staff will handle the full legal due diligence.
+* What is the release model?  Versioning scheme?  Evidence of stability or otherwise of past stable released versions?
+* What is the CI/CD status?  Do explicit code coverage metrics exist? If not, what is the subjective adequacy of automated testing?
+  Do different levels of tests exist (e.g. unit, integration, interface, end-to-end), or is there only partial coverage in this regard?  Why?
+* What licensing restrictions apply?  Again, CNCF staff will handle the full legal due diligence.
+* What are the recommended operational models?  Specifically, how is it operated in a cloud-native environment, such as on Kubernetes?
+
+#### Team
+
+The key high-level questions that the voting TOC members will be looking to have answered are (from the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/):
+
+* Do we believe this is a growing, thriving project with committed contributors?
+* Is it aligned with CNCF's values and mission?
+* Do we believe it could eventually meet the graduation criteria?
+* Should it start at the inception level or incubation level?
+
+Some details that might inform the above include:
+
+* Does the project have a documented process for adding committers?
+* Does it have a documented governance model of any kind?
+* Does it have committers from multiple organizations?
+* Does it have a code of conduct?
+* Does it have a license? Which one? Does it have a CLA or DCO?
+* What is the general quality of informal communication around the project (slack, github issues, PR reviews, technical blog posts, etc)?
+* How much time does the core team commit to the project?
+* How big is the team? Who funds them? Why? How much? For how long?
+* Who are the clear leaders?  Are there any areas lacking clear leadership?  Testing? Release? Documentation?  These roles sometimes go unfilled.
+* Besides the core team, how active is the surrounding community?  Bug reports?  Assistance to newcomers? Blog posts etc.
+* Do they make it easy to contribute to the project? If not, what are the main obstacles?
+* Are there any especially difficult personalities to deal with?  How is this done?  Is it a problem?
+
+#### Users
+
+* Who uses the project?  Get a few in-depth references from 2-4 of them who actually know and understand it.
+* What do real users consider to be it's strengths and weaknesses?  Any concrete examples of these?
+* Perception vs Reality:  Is there lots of buzz, but the software is flaky/untested/unused?  Does it have a bad reputation for some flaw that has already been addressed?
+
+#### Project
+
+* Where does it fit in the market and technical ecosystem?
+* Is it growing or shrinking in that space?  Is that space growing or shrinking?
+* How necessary is it? What do people who don't use this project do?  Why exactly is that not adequate, and in what situations?
+* What are the most important holes? No HA? No flow control? Inadequate integration points?
```

Comment:
I think this question belongs above, in the technical diligence section

Reply:
Agreed, done.